jobserver/backends.py

```diff
@@ -7,6 +7,51 @@
 env = Env()
 
 
+backends = [
+    {
+        "id": 4,
+        "name": "databricks",
+        "slug": "databricks",
+        "is_active": False,
+        "level_4_url": "",
+    },
+    {
+        "id": 1,
+        "name": "EMIS",
+        "slug": "emis",
+        "is_active": True,
+        "level_4_url": "",
+    },
+    {
+        "id": 2,
+        "name": "Expectations",
+        "slug": "expectations",
+        "is_active": False,
+        "level_4_url": "",
+    },
+    {
+        "id": 6,
+        "name": "Graphnet",
+        "slug": "graphnet",
+        "is_active": False,
+        "level_4_url": "",
+    },
+    {
+        "id": 5,
+        "name": "Test",
+        "slug": "test",
+        "is_active": False,
+        "level_4_url": "https://test.opensafely.org",
+    },
+    {
+        "id": 3,
+        "name": "TPP",
+        "slug": "tpp",
+        "is_active": True,
+        "level_4_url": "http://localhost:8001",
+    },
+]
```

Comment:
I can see why these are ordered alphabetically -- but I wonder whether ordering them by ID would make it easier to work out what ID to assign to the next backend when we add more?  Maybe not worth doing now.

Reply:
Yep, that makes much more sense 👍